server/script/migrateAssetsLocation.js

```diff
@@ -0,0 +1,194 @@
+'use strict';
+
+const {
+  Activity,
+  ContentElement,
+  Revision,
+  sequelize
+} = require('../shared/database');
+const {
+  getActivityMetadata,
+  getElementMetadata,
+  SCHEMAS
+} = require('../../config/shared/activities');
+const flatten = require('lodash/flatten');
+const fromPairs = require('lodash/fromPairs');
+const get = require('lodash/get');
+const { Op } = require('sequelize');
+const Promise = require('bluebird');
+const { protocol } = require('../../config/server/storage');
+const storage = require('../shared/storage');
+const toPairs = require('lodash/toPairs');
+
+const regex = /repository\/assets\/(.*)/;
+const ceTypes = ['IMAGE', 'VIDEO', 'AUDIO', 'PDF', 'CAROUSEL', 'MODAL'];
```

Comment:
It should be uppercased. And name should be more precise...`ELEMENT_TYPES` will do the trick.